src/DssSpell.t.sol

```diff
@@ -303,6 +308,33 @@ contract DssSpellTest is DssSpellTestBase {
         );
     }
 
+    function testAllocatorIntegration() public { // add the `skipped` modifier to skip
+        AllocatorVaultLike sparkVault = AllocatorVaultLike(addr.addr("ALLOCATOR_SPARK_A_VAULT"));
+
```

Comment:
nit, We could create another var 
eg.
```
AllocatorIntegrationParams allocatorParams = AllocatorIntegrationParams({
    ilk: "ALLOCATOR-SPARK-A",
    pip: addr.addr("PIP_ALLOCATOR_SPARK_A"),
    registry: addr.addr("ALLOCATOR_REGISTRY"),
    roles: addr.addr("ALLOCATOR_ROLES"),
    buffer: addr.addr("ALLOCATOR_SPARK_A_BUFFER"),
    vault: addr.addr("ALLOCATOR_SPARK_A_VAULT"),
    allocatorProxy: 0x3300f198988e4C9C63F75dF86De36421f06af8c4 // Spark Proxy
})
```
and use for both sanity check & `_checkAllocatorIntegration` .

I think this can help making less typo & things to check.